packages/ag-charts-community/src/chart/series/cartesian/cartesianSeries.ts

```diff
@@ -143,7 +143,7 @@ export interface CartesianSeriesNodeDataContext<
     visible: boolean;
 }
 
-export const RENDER_TO_OFFSCREEN_CANVAS_THRESHOLD = 10;
+export const RENDER_TO_OFFSCREEN_CANVAS_THRESHOLD = 20;
```

Comment:
I was thinking this should be like 1000

Reply:
I went there originally, but backed-off to make a Sparklines-specific change for now.